source/frontend/src/features/properties/map/acquisition/common/update/acquisitionOwners/UpdateAcquisitionOwnersSubForm.tsx

```diff
@@ -18,10 +18,40 @@ import {
 interface IUpdateAcquisitionOwnersSubFormProps {}
 
 const UpdateAcquisitionOwnersSubForm: React.FC<IUpdateAcquisitionOwnersSubFormProps> = () => {
-  const { values } = useFormikContext<WithAcquisitionOwners>();
+  const { values, setFieldValue, handleChange } = useFormikContext<WithAcquisitionOwners>();
   const [removeIndex, setRemoveIndex] = useState<number>(-1);
   const [showRemoveModal, setShowRemoveModal] = useState<boolean>(false);
 
+  const updatePrimaryContacts = (newPrimaryIndex: number) => {
```

Comment:
These functions definitely need test coverage, but I don't see any tests in this PR, are those on the way?

Reply:
Added tests to cover the scenarios for primary contact.